test/manifest.ts

```diff
@@ -283,6 +286,77 @@ describe('Manifest', () => {
       ]);
     });
 
+    it('allows root module to be published, via special "." path', async function () {
```

Comment:
also, I think we should add a test under the `describe('githubRelease'` section. I'd be curious to see what the tag creation looks like for the root package.

also maybe a note somewhere in `docs/manifest-releaser.md` about this root pkg ability?

Reply:
Done 👍

Reply:
Done 👍